tests/drivers/spi/spi_loopback/boards/mimxrt1170_evk_cm7.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright 2023 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+CONFIG_SPI_MCUX_LPSPI_DMA=y
+CONFIG_SPI_ASYNC=y
```

Comment:
I don't think you need to specify this, it's on by default for this test

Reply:
Good catch, this should be fixed now.